platform/android/src/file_source.cpp

```diff
@@ -81,6 +81,13 @@ void FileSource::pause(jni::JNIEnv&) {
     }
 }
 
+jni::jboolean FileSource::isResumed(jni::JNIEnv&) {
+    if (activationCounter) {
+       return  (jboolean) (activationCounter != 0);
```

Comment:
The check here should be `(activationCounter > 0)` otherwise sequence
```
FileSource.activate();
FileSource.deactivate();
FileSource.deactivate();
```
will result in `FileSource.isResumed()` returning `true`.